src/lib/AccountStoreIndexeddb.js

```diff
@@ -22,26 +22,24 @@ class AccountStore {
      * @param {string} dbName
      * @constructor
      */
-    constructor(dbName = 'accounts') {
+    constructor(dbName = AccountStore.ACCOUNT_DATABASE) {
         this._dbName = dbName;
         this._dropped = false;
-        /** @type {IDBDatabase | null} */
-        this._db = null;
+        this._dbPromise = null;
```

Comment:
Not too happy that it doesn't have a type anymore. You probably didn't notice now, but there was most likely no auto-completion anymore when writing `this._dbPromise. ...`. I think it was better when there was a clear variable that had the `IDBDatabase` type.

Reply:
Also, just like in the `KeyStore`, you don't need to initialize it here I think.

Reply:
Well, this._dbPromise. ... is not used anywhere, just await this._dbPromise.
However, readded the type annotation though.

I'm initializing it, to give it the explicit type | null

The reason I replaced this._db by this._dbPromise is that this._dbPromise gets set immediately while this._db only got set after the database was opened. Thus, the approach with this._db lead to problems when simultaneously having multiple connect calls (or any of the other calls that call connect) where it would create multiple database connections but only store the last one. The other ones remain open and prevent the database from being able to be deleted.